src/Ombi/ClientApp/app/requests/movierequests.component.ts

```diff
@@ -149,15 +149,40 @@ export class MovieRequestsComponent implements OnInit {
         event.preventDefault();
     }
 
-    public clearFilter() {
+    public clearFilter(el: any) {
+        el = el.toElement || el.relatedTarget || el.target || el.srcElement;
+
+        el = el.parentElement;
+        el = el.querySelectorAll("INPUT");
+        for (el of el) {
+            el.checked = false;
+            el.parentElement.classList.remove("active");
+        }
+
         this.filterDisplay = false;
         this.filter.availabilityFilter = FilterType.None;
         this.filter.statusFilter = FilterType.None;
         
         this.resetSearch();
     }
 
-    public filterAvailability(filter: FilterType) {
+    public filterAvailability(filter: FilterType, el: any) {
+        el = el.toElement || el.relatedTarget || el.target || el.srcElement;
+
+        el = el.parentElement; //gets radio div
```

Comment:
Can we not move this part of the code into it's own private method? Since it's used below too